mscore/scoreview.h

```diff
@@ -117,6 +117,8 @@ class ScoreView : public QWidget, public MuseScoreView {
 
       //--input state:
       PositionCursor* _cursor;
+      QColor _cursorColor;
+      const int maxCursorAlpha = 220;
```

Comment:
Better name: MAX_CURSOR_ALPHA, to immediately see that it is a constant

Reply:
@igorkorsukov I chose lower-case because the other constant in the same file is like that. Should I change both?

Reply:
Only this one is better.  Others are best fixed with a separate pull request.